src/commands/mod.rs

```diff
@@ -2867,6 +2870,13 @@ fn cmd_restore(
 ) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
     let (from_tree, to_commit);
+    if args.revision.is_some() {
+        return Err(user_error(
+            "`jj restore` does not have a `--revision`/`-r` option. If you'd like to modify\nthe \
+             *current* revision, use `--from`. If you'd like to make a *different* revision,\n \
+             use `--from` or `--changes-in`.",
```

Comment:
This suggests `--from` for both cases. Should the second `--from` be `--to`?

Reply:
Good catch, thanks!